### ISSUE TITLE
MAC Pair searching fails on Ubuntu 20.04

### ISSUE THREAD

**Issue by jcorjaycee** [seannleckie/GrandstreamATAConfigurator#5]
```
===========================
SLE's Grandstream SSH Setup
===========================


Found interface: eth3
Is this the correct interface? (Y/n):
Now scanning your network for a Grandstream device...
Whoops, couldn't get that... 192.168.11.1
System.ComponentModel.Win32Exception (2): No such file or directory
   at System.Diagnostics.Process.ForkAndExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at GrandstreamATAConfigurator.Program.GetMacIpPairs()+MoveNext() in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 611
   at GrandstreamATAConfigurator.Program.GetMacByIp(String ip) in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 589
   at GrandstreamATAConfigurator.Program.PortScan() in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 552
Unhandled exception. System.ComponentModel.Win32Exception (2): No such file or directory
   at System.Diagnostics.Process.ForkAndExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at GrandstreamATAConfigurator.Program.GetMacIpPairs()+MoveNext() in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 611
   at GrandstreamATAConfigurator.Program.GetMacByIp(String ip) in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 589
   at GrandstreamATAConfigurator.Program.PortScan() in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 552
   at GrandstreamATAConfigurator.Program.Main() in C:\Users\jared\RiderProjects\GrandstreamATAConfigurator\GrandstreamATAConfigurator\Program.cs:line 86
Aborted (core dumped)
```

**Comment by jcorjaycee** [comment]
I need to replicate this on Manjaro as this was run in WSL. From recent testing, I recognize 192.168.11.1 as an anomalous gateway in my interface settings which may have been fixed by b9d711ea8b3d811c8ad13d242d672165f2a3c00d

**Comment by jcorjaycee** [comment]
```
=====================
Grandstream SSH Setup
=====================


Found interface: enp3s0f1
Is this the correct interface? (Y/n): 
Now scanning your network for a Grandstream device... Please wait.
Whoops, couldn't get that... 192.168.69.1
System.AggregateException: One or more errors occurred. (Connection refused)
 ---> System.Net.Sockets.SocketException (111): Connection refused
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at GrandstreamATAConfigurator.NetworkUtils.PortScan(String ip, String& newIp)
Unhandled exception. System.AggregateException: One or more errors occurred. (Connection refused)
 ---> System.Net.Sockets.SocketException (111): Connection refused
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at System.Net.Sockets.TcpClient.CompleteConnectAsync(Task task)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at GrandstreamATAConfigurator.NetworkUtils.PortScan(String ip, String& newIp)
   at GrandstreamATAConfigurator.Program.Main()
Aborted (core dumped)
```

**Comment by jcorjaycee** [comment]
This thing really needs a try/catch?

**Comment by jcorjaycee** [comment]
Issue partially resolved by c7323094ed2f0b1afeb4e7a118649250075c5593
Main issue is regex in NetworkUtils.GetMacIpPairs
This regex was designed for Windows `arp`, not *nix `arp`
Let's fix that!
